src/mono/System.Private.CoreLib/src/System/Threading/Overlapped.cs

```diff
@@ -84,8 +84,7 @@ internal sealed unsafe class OverlappedData
                     }
                 }
 
-                //CORERT: NativeOverlapped* pNativeOverlapped = (NativeOverlapped*)Interop.MemAlloc((UIntPtr)(sizeof(NativeOverlapped) + sizeof(GCHandle)));
-                NativeOverlapped* pNativeOverlapped = (NativeOverlapped*)Marshal.AllocHGlobal((IntPtr)(sizeof(NativeOverlapped) + sizeof(GCHandle)));
+                NativeOverlapped* pNativeOverlapped = (NativeOverlapped*)NativeMemory.Alloc((nuint)(sizeof(NativeOverlapped) + sizeof(GCHandle)));
```

Comment:
I have converted several other places that used `Marshal.AllocHGlobal` to `NativeMemory.Alloc` while I was on it.